testfiles/RadLoHydrHeatCoolAuto.idf

```diff
@@ -2,7 +2,7 @@
 ! Basic file description:  1 story building divided into 3 interior conditioned zones. Roof with no plenum.
 !                          No ground contact with floor.
 !
-! Highlights:              Basic test of the hydronic radiant heating and cooling system with autosizing
+! Highlights:              Basic test of the hydronic radiant heating and cooling system with scalable and autosizing
```

Comment:
@nealkruis The example file demonstrates two scalable sizing methods (CapacityPerFloorArea
and FractionOfAutosizedHeatingCapacity).